src/main/kotlin/com/example/flashcardbackend/deck/DeckRepository.kt

```diff
@@ -0,0 +1,88 @@
+package com.example.flashcardbackend.deck
+
+import com.example.flashcardbackend.flashcard.FlashcardListItem
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.dao.DataAccessException
+import org.springframework.jdbc.core.JdbcTemplate
+import org.springframework.jdbc.core.ResultSetExtractor
+import org.springframework.stereotype.Repository
+import org.springframework.transaction.annotation.Transactional
+import java.sql.ResultSet
+import java.sql.SQLException
+
+@Transactional
```

Comment:
This is fine, but my OCD tells me that these annotations must be in this order: @Repository @Transactional 🙈

Reply:
I should add ktlint to the project